### PR TITLE
CU-8669zb648 | Rework Bottom Bar to Match new Figma Designs

### PR DESCRIPTION
**Pull request by Jermainelr**
LibraryIcon
HomeIcon
MarketIcon
WalletIcon
SearchIcon

### CODE REVIEW

**Comment by newmskywalker** [comment]
Task linked: [CU-8669zb648 Rework Bottom Bar to Match new Figma Designs](https://app.clickup.com/t/8669zb648)

File: android/app-newm/src/main/java/io/newm/NewmAppComposable.kt [hunk 1]
```diff
@@ -144,16 +178,46 @@ fun BottomNavigationBarPreview() {
     NewmBottomNavigation(Screen.HomeRoot, true) {}
 }
 
+// Based on content from: https://github.com/wlara/android-next-gen/blob/main/app/src/main/java/com/github/wlara/nextgen/ui/home/HomeScreen.kt
```
Comment (cristhianescobar) [inline android/app-newm/src/main/java/io/newm/NewmAppComposable.kt:182]: 👍🏼

File: android/app-newm/src/main/java/io/newm/NewmAppComposable.kt [hunk 2]
```diff
@@ -179,4 +243,13 @@ val routesWithoutBottomNavBar: List<String> by lazy {
...
         Screen.NowPlayingScreen.route
     )
-}
\ No newline at end of file
+}
+
+private fun iconGradient(
+    startColor: Color,
+    endColor: Color
+) = Brush.linearGradient(
```
Comment (cristhianescobar) [inline android/app-newm/src/main/java/io/newm/NewmAppComposable.kt:252]: Pretty neat solution! 👨🏼‍🍳

**Review by cristhianescobar — APPROVED** [review]
LGTM

**Pull request merged**